core/templates/css/oppia.css

```diff
@@ -1881,6 +1881,22 @@ pre.oppia-pre-wrapped-text {
 .popover-content {
   padding: 10px;
 }
+
+.popover-content:after {
+  background-color: rgb(255,255,255);
+  border-bottom: 1px solid rgba(0,0,0,0.2);
+  border-left: 1px solid rgba(0,0,0,0.2);
+  content: '';
+  height: 16px;
+  left: 0%;
+  margin-left: -8px;
+  margin-top: 8px;
+  position: absolute;
+  top: 0%;
+  transform: rotate(45deg);
+  width: 16px;
+}
+
```

Comment:
Is this something that is very commonly used across the codebase or just in one or two places?